samples/net/azure_iot_hub/README.rst

```diff
@@ -162,15 +165,20 @@ If a new telemetry interval is set in the device twin, the console output is lik
 Dependencies
 ************
 
-This sample uses the following |NCS| libraries and drivers:
+The sample uses the following |NCS| and Zephyr libraries:
 
 * :ref:`lib_azure_iot_hub`
-* :ref:`lte_lc_readme`
+* :ref:`net_if_interface`
+* :ref:`net_mgmt_interface`
 
-It uses the following `sdk-nrfxlib`_ library:
+It uses the following libraries and secure firmware component for nRF91 series builds:
 
-* :ref:`nrfxlib:nrf_modem`
+* :ref:`nrf_modem_lib_readme`
+* :ref:`Trusted Firmware-M <ug_tfm>`
+* :ref:`modem_info_readme`
 
-In addition, it uses the following secure firmware component:
+It uses the following libraries for nRF7 series builds:
 
-* :ref:`Trusted Firmware-M <ug_tfm>`
+* :ref:`nrf_security`
```

Comment:
None of these are called directly

Reply:
I'm not really sure what to put in these lists..